src/app/components/TwitCreate.tsx

```diff
@@ -40,22 +47,16 @@ export default function TwitCreate() {
     }
   }
 
-  const mockData = {
-    avatar:
-      'https://upload.wikimedia.org/wikipedia/commons/thumb/4/49/Flag_of_Ukraine.svg/375px-Flag_of_Ukraine.svg.png',
-  }
+  const avatar =
```

Comment:
Use as default value in `Image` component or completely remove. Soon we can get it from backend